src/web/frontend/components/Header/UserNav/UserNav.tsx

```diff
@@ -13,7 +13,7 @@ function UserNav() {
     return (
         <UserProvider.context.Consumer>{context => (
             <React.Fragment>
-                {context.user && context.user.authenticated
```

Comment:
should keep `context.user && context.user.authenticated`

Reply:
done